includes/gateways/stripe/includes/admin/admin-helpers.php

```diff
@@ -204,3 +179,22 @@ function give_stripe_connect_delete_options() {
 	give_delete_option( 'live_publishable_key' );
 	give_delete_option( 'test_publishable_key' );
 }
+
+/**
+ * This helper function is used to convert slug into names of Stripe connection.
```

Comment:
@mehul0810 this function return a single name, so fix typo in the description.
```suggestion
 * This helper function is used to convert slug into the name of Stripe connection.
```